target/linux/ath79/dts/qca9531_yuncore_cpe830.dts

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "yuncore,cpe830", "qca,qca9531";
+	model = "YunCore CPE830";
+
+	chosen {
+		bootargs = "console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd";
+	};
+
+	keys {  
+		compatible = "gpio-keys";
+
+		reset { 
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	aliases {
+		led-boot = &status;
+		led-failsafe = &status;
+		led-running = &status;
+		led-upgrade = &status;
+	};
+
+	leds {
```

Comment:
@grosjo please, compare your board GPIO mapping with CPE830 in ar71xx:
https://github.com/openwrt/openwrt/blob/master/target/linux/ar71xx/files/arch/mips/ath79/mach-ap90q.c#L67

It's different.

Reply:
no it is not different. Can you please satte exactly what do you want ?

Reply:
```
status: status {
	label = "cpe830:green:status";
	gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
};
```

Reply:
done

Reply:
So, you just removed status LED, added LEDs from ar71xx code... have you tested your code on the real hardware? I have some serious doubts here...

Reply:
of course not, I am repeating for so may time : I will do it when I reach my second home. WHy don't you read the comments before arguing

Reply:
Then make changes in code **after** they are confirmed and tested, not before.

Reply:
of course not, otherwise, how can i test it ? :D

Reply:
Make changes locally, test them on the hardware, then if everything is fine, push changes to your PR. Otherwise, you are just wasting our time as people following this PR get notifications every time...

Reply:
![Screenshot from 2019-08-16 15-19-47](https://user-images.githubusercontent.com/2287111/63167235-539d5700-c039-11e9-93af-dd73f1714102.png)
What do you look for in adding this tag ?
Can you be much more positive and finger point the changes you require instead of arguing in the clouds ?

Reply:
I'm not arguing with you. There are some basic rules and standards people try to follow to make the project ongoing and in good shape. Asking developers for reviewing code which hasn't been tested on real hardware, changing it several times within 15 minutes period just because someone pointed out that you are probably mixing different devices... is just rude. You wasted a lot of my time asking for review and point out your mistakes which turned out to be just a copy&paste of a broken code.

I assigned the label and in my opinion, after ~200 comments here, for PR with support for a very simple device under DTS-enabled target, **is valid**. I hope you can do better.

Reply:
Not interested in your insults